### ISSUE TITLE
Is there any way for end user to control size of plots?

### ISSUE THREAD

**Issue by rpruim** [rstudio/learnr#85]
This one might be trickier.

The creator of a tutorial can use `fig.height` to allocate space.  But the amount of space desired might depend on the plot the user makes.  Is there any way to make that something the user of the tutorial can adjust?

**Comment by jjallaire** [comment]
Yeah that is much trickier and not something we can probably implement in
the short term.

**Comment by rpruim** [comment]
That was my guess.  But perhaps it can go on the wish list so folks can think about what it would take to make that possible.
